src/store/annotations/util.ts

```diff
@@ -0,0 +1,31 @@
+import * as uuid from 'uuid';
+import { Annotation } from '../../@types';
+
+export function addClientIds(annotation: Annotation): Annotation {
+    if (annotation.target.type !== 'drawing') {
+        return annotation;
+    }
+
+    const { target } = annotation;
+    const { path_groups: pathGroups } = target;
+
+    return {
+        ...annotation,
+        target: {
+            ...annotation.target,
+            // eslint-disable-next-line @typescript-eslint/camelcase
+            path_groups: pathGroups.map(pathGroup => {
+                const { paths } = pathGroup;
```

Comment:
Can this this be inlined with the function definition such as `.map(({ paths, ...rest }) => {`